postgres/data.go

```diff
@@ -177,6 +180,15 @@ func convInt64(val string) (int64, error) {
 	return i, err
 }
 
+func convNumeric(val string) (*big.Rat, error) {
+	r := new(big.Rat)
+	_, ok := r.SetString(val)
+	if !ok {
+		return r, fmt.Errorf("can't convert to big.Rat")
+	}
+	return r, nil
+}
```

Comment:
Looking forward to get rid of this kind of duplication soon after the pending PRs are merged :) - same comments apply though here as the one in mysql/data.go

Reply:
Done.
Yes, agreed.